templates/zerver/help/getting-started-with-zulip.md

```diff
@@ -0,0 +1,58 @@
+# Getting started with Zulip
+
+Welcome to Zulip!  Like any new tool, mastering Zulip will take some time,
+but once you get the hang of it, you'll never want to use a chatroom tool
+again.
+
+Zulip is designed around a few key ideas:
+
+- You spend far more time reading communications than writing them.  So
+  Zulip is primarily designed to optimize consuming messages; either reading
+  them, or making an informed decision not to read them.
+- Messages in Zulip are organized into streams and topics.  Streams are
+  similar to chatrooms, IRC channels, or email lists, and determine who
+  receives the message.  Each conversation in a stream has a topic, which
+  has the same role as the subject line of an email (though topics are
+  usually shorter, e.g. "logo" or "logo design", not "feedback on the new
+  logo design?").
+- Zulip keeps careful track of which messages and threads you've read, and
+  always places you exactly where you left off.
```

Comment:
We introduce the word "threads" here without clarifying that threads=topics; maybe we could insert something above to make the equivalence explicit.  e.g.

Each conversation in a stream has a topic, which has the same role as the subject line of an email (though topics are usually shorter, e.g. "logo" or "logo design", not "feedback on the new logo design?") in that it organizes messages into threads.